packages/styletron-react/src/__tests__/tests.browser.tsx

```diff
@@ -49,22 +46,20 @@ test("styled (static)", t => {
     </Provider>,
   );
   const divs = wrapper.find("div");
-  t.equal(divs.length, 1, "single div rendered");
-  t.ok(divs.hasClass("foo bar"), "explicit and generated class names merged");
-  t.end();
+  expect(divs.length).toBe(1);
+  expect(divs.hasClass("foo bar")).toBe(true);
 });
 
-test("styled (dynamic)", t => {
-  t.plan(1);
-  const Widget = styled("div", (props: {$foo: boolean}) => ({
+test("styled (dynamic)", () => {
+  const Widget = styled<"div", {$foo: boolean}>("div", props => ({
```

Comment:
will this sort of type change need to be done in downstream code? (moving arg position type to generic param and specifying element type)

Reply:
not sure why this change was made… - having it in function argument will work just fine in this case